lib/src/session.rs

```diff
@@ -252,6 +252,23 @@ impl Session {
             .ok_or(HandleError::InvalidBodyHandle(handle))
     }
 
+    /// Drop a [`Body`][body] from the [`Session`], given its [`BodyHandle`][handle].
```

Comment:
i think we can use intradoc links here :slightly_smiling_face:

Reply:
Oh right I had just copied from what we had before let me fix that up